irohad/model/converters/pb_query_response_factory.hpp

```diff
@@ -18,18 +18,19 @@
 #ifndef IROHA_PB_QUERY_RESPONSE_FACTORY_HPP
 #define IROHA_PB_QUERY_RESPONSE_FACTORY_HPP
 
-#include <responses.pb.h>
-#include <model/account_asset.hpp>
+#include <nonstd/optional.hpp>
+
+#include "model/account_asset.hpp"
 #include "model/common.hpp"
 #include "model/queries/responses/account_assets_response.hpp"
 #include "model/queries/responses/account_detail_response.hpp"
 #include "model/queries/responses/account_response.hpp"
-#include <nonstd/optional.hpp>
+#include "model/queries/responses/asset_response.hpp"
 #include "model/queries/responses/error_response.hpp"
+#include "model/queries/responses/roles_response.hpp"
 #include "model/queries/responses/signatories_response.hpp"
 #include "model/queries/responses/transactions_response.hpp"
-#include "model/queries/responses/roles_response.hpp"
-#include "model/queries/responses/asset_response.hpp"
+#include "responses.pb.h"
```

Comment:
IMO it is better to create single header `model/responses.hpp`, which includes all these responses. And then, include it here and in other places.

Reply:
This is done in shared model in some sense, which will replace this code in near future.